Add image decoration (#5052)
qgis/QGIS-Documentation: docs/user_manual/introduction/general_tools.rst

Grid
....

> |addGrid| :menuselection:`Grid` allows you to add a coordinate grid and coordinate

Author: I can bring back the full descriptions, but I prefer to do that in the next update and not online. I thought more concise documentation was preferred. About the :sup: I was sure it was not the right style because that is used for toolbar icons, so I thought :Menuselection: was the best option. I have ofcourse rebuild the documentation to view the result and the result looked great. But with :guilabel: I am sure it will look great too! 
I was surprised how much you two have found. Great job! Next time I will bring in 'smaller' changes.

annotations to the map canvas.

> #. Select menu option :menuselection:`View --> Decorations --> Grid...` to open the dialog.

Reviewer: The three dots/periods appear in the menu in the GUI, so I think they should a part of the :guilabel: in the documentation.

> #. Select menu option :menuselection:`View --> Decorations --> Grid...` to open the dialog.

Author: actually I prefer to keep it as suggested. The icon is a nice visual start for each decoration and in the menu selection it is fine to leave it out. Several years ago I have introduced it, and I still like this little chapter.

      The Title Decoration Dialog

> #. Make sure |checkbox| :guilabel:`Enable Title Label`) is checked

Reviewer: ```suggestion
#. Make sure |checkbox| :guilabel:`Enable Title Label` is checked
```

   <text_format>` options. Quickly set the font color and opacity by clicking
   the black arrow to the right of the font combo box.
#. Select the :ref:`color <color-selector>` to apply to the title's
   :guilabel:`Background bar color`.
#. Choose the :guilabel:`Placement` of the label in the canvas: options are

>    :guilabel:`Top left`, :guilabel:`Top Center` (default for Title Label decoration),

Reviewer 2: ```suggestion
   :guilabel:`Top left`, :guilabel:`Top Center` (default),
```
left over from the split?

Copyright Label
...............

|copyrightLabel| :menuselection:`Copyright Label` can be used to decorate your map with a **Copyright** label.

> The Copyright Label dialog is almost equal to the dialog of :ref:`title_label_decoration`.

Reviewer: ```suggestion
The Copyright Label dialog is almost identical to the :ref:`title_label_decoration` dialog.
```

Copyright Label
...............

> |copyrightLabel| :guilabel:`Copyright Label` can be used to decorate your map with a **Copyright** label.

Reviewer: ```suggestion
|copyrightLabel| :guilabel:`Copyright Label` can be used to decorate your
map with a **Copyright** label.
```
We try to avoid very long lines. 70 to 80 characters is a reasonable maximum.

      The Copyright Decoration Dialog

.. index:: Image decoration
.. _image_decoration:

Image Decoration
................

|addImage| :menuselection:`Image` allows you to add an image (logo, legend, ..) on the map canvas.

To add an image:

#. Select menu option :menuselection:`View --> Decorations --> Image...` to open the dialog.

   .. _figure_image_decoration:

   .. figure:: img/image_decoration.png
      :align: center

      The Image Decoration Dialog

#. Make sure |checkbox| :guilabel:`Enable Image` is checked.
#. Select a bitmap (i.e. png or jpg) or SVG image using the browse button.

> #. If you have choosen a parameter enabled SVG than you can also set a :guilabel:`Fill` or :guilabel:`Stroke` color. With :guilabel:`Stroke` color you can set the outline color of the SVG image.

Author: outline should help Dutch translator to chose from: hersenbloeding, beroerte, apoplexie, attaque, hartinfarct, hartaanval, hartverlamming, infarct, handslag, herseninfarct, kwaststreek, knuffelen, liefkozen,  harde klap, klap van het noodlot, zware klap, aaien, strelen, beroerte, apoplexie, flap, houw, klap, mep, slag, gevecht, kamp, strijd, treffen, veldslag, schop, stoot, tik, klets, klop, veeg, beweging, zet, lijn, streep, streek, schreef, band, reep, strook, straal, scheiding, haarscheiding, haal, schrap, wapenbalk

#. Click :guilabel:`Apply` to verify that it looks as expected or
   :guilabel:`OK` if you're satisfied.

.. index:: Image decoration
.. _image_decoration:

Image Decoration
................

|addImage| :guilabel:`Image` allows you to add an image (logo, legend, ..) on the map canvas.

To add an image:

#. Select menu option :menuselection:`View --> Decorations --> Image...`
   to open the dialog.

   .. _figure_image_decoration:

   .. figure:: img/image_decoration.png
      :align: center

      The Image Decoration Dialog

#. Make sure |checkbox| :guilabel:`Enable Image` is checked

> #. Select a bitmap (e.g. png or jpg) or SVG image using the :guilabel:`...` :sup:`Browse` button

Reviewer: ```suggestion
#. Select a bitmap (e.g. png or jpg) or SVG image using the
   :guilabel:`...` :sup:`Browse` button
```

#. If you have chosen a parameter enabled SVG then you can also set
   a :guilabel:`Fill` or :guilabel:`Stroke` (outline) color.
   For bitmap images, the color settings are disabled.
#. Set a :guilabel:`Size` of the image in mm. The width of selected image is used to
   resize it to given :guilabel:`Size`.
#. Choose where you want to place the image on the map canvas with the
   :guilabel:`Placement` combo box. The default position is :guilabel:`Top Left`.
#. Set the :guilabel:`Horizontal` and :guilabel:`Vertical Margin from (Canvas) Edge`.
   These values can be set in **Millimeters**, **Pixels** or as a **Percentage** of
   the width or height of the map canvas.
#. Click :guilabel:`Apply` to verify that it looks as expected and :guilabel:`OK`
   if you're satisfied.

.. index:: North arrow
.. _northarrow_decoration:

North Arrow
...........

|northArrow| :guilabel:`North Arrow` allows you to add a north arrow on the map canvas.

To add a north arrow:

> #. Select menu option :menuselection:`View --> Decorations --> North Arrow...` to open the dialog.

Reviewer: ```suggestion
#. Select menu option :menuselection:`View --> Decorations --> North Arrow...`
   to open the dialog.
```

      The Scale Bar Dialog

> #. Make sure |checkbox| :guilabel:`Enable scale bar` is checked.

Reviewer: ```suggestion
#. Make sure |checkbox| :guilabel:`Enable scale bar` is checked
```
Current policy is to not use periods for simple sentences in list items. Simple means no commas, periods, ... in the list item.

Author: I found out many sentences of list items ended with periods and others did not. So I added periods. I have no problem following the current policy! Thanks for informing me!

Reviewer: FYI: The current policy on lists is documented in docs/documentation_guidelines/writing.rst - https://docs.qgis.org/testing/en/docs/documentation_guidelines/writing.html#lists

#. Choose a style from the :guilabel:`Scale bar style` |selectString| combo box

> #. Select the :guilabel:`Color of bar` |selectColor| by choosing.

Reviewer: ```suggestion
#. Select the :guilabel:`Color of bar` |selectColor| by choosing
```

   a fill color (default: black) and an outline color (default: white). The scale
   bar fill and outline can be made opaque by clicking on the down arrow to the right
   of the color input.

> #. Select the font for the scale bar from the :guilabel:`Font of bar` |selectString| combo box.

Reviewer: ```suggestion
#. Select the font for the scale bar from the :guilabel:`Font of bar` |selectString| combo box
```

> #. Set the :guilabel:`Size of bar` |selectNumber|.

Reviewer: ```suggestion
#. Set the :guilabel:`Size of bar` |selectNumber|
```

#. Optionally check |checkbox| :guilabel:`Automatically snap to round number

>    on resize` to display easy-to-read values.

Reviewer: ```suggestion
   on resize` to display easy-to-read values
```

To add a scale bar:

> #. Select menu option :menuselection:`View --> Decorations --> Scale Bar...` to open the dialog

Reviewer: ```suggestion
#. Select menu option :menuselection:`View --> Decorations --> Scale Bar...`
   to open the dialog
```

      The Layout Extents Dialog

#. Make sure |checkbox| :guilabel:`Show layout extents` is checked.
#. Optionally change the symbol and labeling of the extents.

> #. Click :guilabel:`Apply` to verify that it looks as expected and :guilabel:`OK` if you're satisfied.

Reviewer: ```suggestion
#. Click :guilabel:`Apply` to verify that it looks as expected and :guilabel:`OK`
   if you're satisfied.
```
